scripts/e2e/opentelemetry-logs/compose.yaml

```
@@ -1,17 +1,7 @@
 name: opentelemetry-vector-e2e
 services:
```

```suggestion
services:
  cleanup-logs:
    image: busybox
    command: [ "sh", "-c", "rm -f /tmp/*.log && touch /cleanup-done" ]
    volumes:
      - type: bind
        source: ../../../tests/data/e2e/opentelemetry/logs/output
        target: /tmp
    healthcheck:
      test: ["CMD-SHELL", "file /cleanup-done"]
      interval: 1s
      retries: 10
      timeout: 1s
```

I think you can keep the cleanup-logs service if you implement something like this (have not tested)

---

Nice! I will keep this pattern in mind for the future.